Moderation_Matrix.md

```diff
@@ -0,0 +1,52 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
+  - Drug related conversation (Rule 7)
+  - Special attention to treating everyone with respect
+  - Mental health issues discussion (Rule 12)
+- A second infraction of the following:
```

Comment:
Since 1 zap ouchies are minor by definition, we should consider applying the 0 zap second infraction rule to all of them instead of just these ones. 

This is simpler, more consistent and slightly friendlier to otherwise well meaning and / or anxious newbs.

If every innocuous offense receives a zap immediately there's a risk that it discourages engagement / participation by new people, many of whom are already hesitant / apologetic when asking their first questions

Reply:
I'm not sure if they should all be 0 zaps, but I do like the idea of having these be a list of 0 zaps, because I think the wording "A second infraction of the following" can be somewhat confusing to people. Especially since how a second infraction of all of them will be given an increase of zaps for a second infraction. 

I think we have to zap people who mini mod, Drug related conversation, and not treating people with respect.

Reply:
I also would support a 0 zap category for these items.